### ISSUE TITLE
Lammpstrj bounding box origin (shifts)

### ISSUE THREAD

**Issue by PandaCodes** [mdtraj/mdtraj#1814]
Hi there! Thank you for maintaining this great project.

My problem is the following:
When loading a Trajectory from the `.lammpstrj` file it is impossible to retrieve original coordinates if they are given in the "scaled" format (being [default](https://docs.lammps.org/dump.html)), which means they have a value in between 0 and 1 in units of a simulation box boundaries. This is because the "origin" point (which represents the shift) of the simulation box is not stored anywhere. 
I can see that  box lengths and angles get stored in the `unitcell_lengths` and `unitcell_angles` correspondingly (which I am not quite sure if it is correct as well).

I would like to make a PR, though am not sure about the best way to fix that:
- In case this sounds like a common story for a `Trajectory`, maybe something like `unitcell_shifts` property could be added, although I am not aware of how to handle this thing in the already existing logic of "unitcells" (I didn't find a particular documentation on this entity)
- Otherwise, one could  "unscale" coordinates if necessary using shifts, lengths & angles in-place during the file parsing. This might be back-incompatible for some usecases though.

**Comment by mattwthompson** [comment]
Unscaled may be the default but I'd definitely say it's non-standard within the field. To my knowledge, LAMMPS is the only engine that writes out files like that as some sort of default.

We'd likely accept a PR that supports this sort of behavior with an optional toggle, but my $0.02 is that writing coordinates with distance units is an easier way to go.

**Comment by sukritsingh** [comment]
Yeah to be honest I'm not sure how ubiquitous this kind of use is outside LAMMPS - most other engines usually scale because it's more about relative coordinates/features anyways. 

> I can see that box lengths and angles get stored in the unitcell_lengths and unitcell_angles correspondingly (which I am not quite sure if it is correct as well).

I'm open to potentially accepting a PR with the toggle but depending on, your need there may be another function that grants you what you ultimately need
